### PR TITLE
Add a md file for Environment (Angular)

### LINKED ISSUE #4991: Document how to use remote environment

### PR DESCRIPTION
**Pull request by bnymncoskuner**
Resolves #4991

### CODE REVIEW

**Comment by hikalkan** [comment]
I assigned to 3.1. Right?

File: docs/en/UI/Angular/Environment.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Environment
+
+Every application needs some ** environment ** variables. In Angular world, this is usually managed by `environment.ts`, `environment.prod.ts` and so on. It is the same for Abp as well.
```
Comment (mehmet-erim) [inline docs/en/UI/Angular/Environment.md]:
```suggestion
Every application needs some ** environment ** variables. In Angular world, this is usually managed by `environment.ts`, `environment.prod.ts` and so on. It is the same for ABP as well. 
```

File: docs/en/UI/Angular/Environment.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+```typescript
+export interface Environment {
+  apis: Apis;
+  application: Application;
+  hmr?: boolean;
+  localization?: { defaultResourceName?: string };
+  oAuthConfig: AuthConfig;
+  production: boolean;
+  remoteEnv?: RemoteEnv;
+}
```
Comment (mehmet-erim) [inline docs/en/UI/Angular/Environment.md]:
```suggestion
export interface Environment {
  apis: Apis;
  application: Application;
  oAuthConfig: AuthConfig;
  production: boolean;
  remoteEnv?: RemoteEnv;
}
```

`hmr` and `localization` properties will probably be removed from the type.

File: docs/en/UI/Angular/Environment.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+* `name`: Name of the backend Application. It is also used by `logo.component` if `logoUrl` is not provided.
+* `logoUrl`: Url of the application logo. It is used by `logo.component`
+* `baseUrl`: [For detailed information](./Multi-Tenancy.md#domain-tenant-resolver)
+
+## Localization
+
+You can read about `Localization` [here in detail](./Localization.md)
```
Comment (mehmet-erim) [inline docs/en/UI/Angular/Environment.md]:
```suggestion
```

For the reason I explained above, this title should never exist.

File: docs/en/UI/Angular/Multi-Tenancy.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ On the page above, you can;
 You can switch between existing tenants by using the tenant switching component in the child pages of the `AccountLayoutComponent` (like Login page). Angular UI sends the selected tenant id to the backend as `__tenant` header on each request.
 
 
-## Domain Tenant Resolver
+<h2 id="domain-tenant-resolver">Domain Tenant Resolver</h2>
```
Comment (armanozak) [inline docs/en/UI/Angular/Multi-Tenancy.md]:
```suggestion
## Domain Tenant Resolver
```

**Review by armanozak — CHANGES_REQUESTED** [review]

**Review by armanozak — APPROVED** [review]

**Pull request merged**